src/Path.php

```diff
@@ -340,4 +340,49 @@ public static function find($paths, $file)
 		// Could not find the file in the set of paths
 		return false;
 	}
+
+	/**
+	 * Resolves /./, /../ and multiple / in a string and returns the resulting absolute path, inspired by Flysystem
+	 * Removes trailing slashes
+	 *
+	 * @param   mixed   $path   A path to resolve
```

Comment:
```suggestion
	 * @param   string   $path   A path to resolve
```

Unless there is a reason this should support more types?